### PR TITLE
Adding OpenLDAP & Docs

### LINKED ISSUE #1170: Requesting LDAP
# Feature request

**Is your feature request related to a problem? Please describe.**
it would be nice to have a LDAP system built in to the list for easier install

**Describe the solution you'd like**
add some form of LDAP container like linuxserver/ldap-auth

### PR DESCRIPTION
**Pull request by angrycuban13**
# Pull request

Closes #1170

### CODE REVIEW

File: compose/.apps/openldap/openldap.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      driver: json-file
+      options:
+        max-file: ${DOCKERLOGGING_MAXFILE}
+        max-size: ${DOCKERLOGGING_MAXSIZE}
+    restart: unless-stopped
+    volumes:
+      - /etc/localtime:/etc/localtime:ro
+      - ${DOCKERCONFDIR}/openldap/database:/var/lib/ldap
+      - ${DOCKERCONFDIR}/openldap/config:/etc/ldap/
+      - ${DOCKERSHAREDDIR}:/shared
```
Comment (nemchik) [inline compose/.apps/openldap/openldap.yml:27]: don't include shared on new apps

**Review by nemchik — CHANGES_REQUESTED** [review]

File: compose/.apps/openldap/openldap.yml (hunk 1 above)
Comment (angrycuban13) [reply compose/.apps/openldap/openldap.yml]: 576c187

**Review by nemchik — APPROVED** [review]

**Comment by nemchik** [comment]
One failing check (navigation link in mkdocs) and I think we're good.

File: docs/apps/openldap.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# OpenLDAP
+
+[![Docker Pulls](https://img.shields.io/docker/pulls/linuxserver/duplicati?style=flat-square&color=607D8B&label=docker%20pulls&logo=docker)](https://hub.docker.com/r/osixia/openldap/)
```
Comment (nemchik) [inline docs/apps/openldap.md:3]: Duplicati?

File: docs/apps/openldap.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+[![GitHub Stars](https://img.shields.io/github/stars/linuxserver/docker-ombi?style=flat-square&color=607D8B&label=github%20stars&logo=github)](https://github.com/docker-openldap)
```
Comment (nemchik) [inline docs/apps/openldap.md:4]: Ombi?

**Review by nemchik — CHANGES_REQUESTED** [review]

File: docs/apps/openldap.md (hunk 2 above)
Comment (angrycuban13) [reply docs/apps/openldap.md]: Talk about a bad search and replace.... 8760ab0

**Review by nemchik — APPROVED** [review]

**Pull request merged**